lms/services/lti_grading/_v13.py

```diff
@@ -66,7 +66,7 @@ def record_result(self, grading_id, score=None, pre_record_hook=None):
                 scopes=self.LTIA_SCOPES,
                 json=payload,
                 headers={"Content-Type": "application/vnd.ims.lis.v1.score+json"},
-            ).json()
```

Comment:
We don't rely on this value outside, json looked better than a `Response` but it's incorrect.

We can think about a better return value after this is fixed.